1714945016629	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms. (why=schedule)
1714947435037	Sync.SyncScheduler	DEBUG	Going to sleep, doing a quick sync.
1714947435038	Sync.Service	DEBUG	User-Agent: Firefox/125.0.3 (Linux x86_64) FxSync/1.127.0.20240503195641.desktop
1714947435038	Sync.Service	INFO	Starting sync at 2024-05-05 23:17:15 in browser session xh_alQ_XLh5n
1714947435038	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1714947435042	Sync.Status	INFO	Resetting Status.
1714947435042	Sync.Status	DEBUG	Status.service: success.status_ok => success.status_ok
1714947435042	FirefoxAccounts	DEBUG	FxAccountsProfileClient: Requested profile
1714947435043	FirefoxAccounts	DEBUG	getOAuthToken enter
1714947435043	FirefoxAccounts	TRACE	getCachedToken returning cached token
1714947435043	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1714947435043	Services.Common.RESTRequest	DEBUG	GET request to https://profile.accounts.firefox.com/v1/profile
1714947435044	Sync.SyncAuthManager	INFO	Getting sync key
1714947435044	FirefoxAccounts	DEBUG	getOAuthToken enter
1714947435045	FirefoxAccounts	TRACE	getCachedToken returning cached token
1714947435045	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1714947435045	Sync.SyncAuthManager	INFO	Getting a sync token from: https://token.services.mozilla.com/1.0/sync/1.5
1714947435045	Sync.SyncAuthManager	DEBUG	Getting a token using OAuth
1714947435045	Services.Common.TokenServerClient	DEBUG	Beginning OAuth token exchange: https://token.services.mozilla.com/1.0/sync/1.5
1714947435045	Services.Common.RESTRequest	DEBUG	GET request to https://token.services.mozilla.com/1.0/sync/1.5
1714947444108	Sync.SyncScheduler	DEBUG	Network link status changed to "down". Offline?: false
1714947444109	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1714947444110	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms. (why=schedule)
1714947444111	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1714947444137	Services.Common.RESTRequest	DEBUG	GET https://token.services.mozilla.com/1.0/sync/1.5 failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1714947444138	Sync.SyncAuthManager	ERROR	Non-authentication error in _fetchTokenForUser: TokenServerClientNetworkError({"error":{}})(resource://services-common/tokenserverclient.sys.mjs:28:36) JS Stack trace: TokenServerClientNetworkError@tokenserverclient.sys.mjs:51:16
_tokenServerExchangeRequest@tokenserverclient.sys.mjs:235:13
1714947444138	Sync.Status	DEBUG	Status.login: success.login => error.login.reason.network
1714947444138	Sync.Status	DEBUG	Status.service: success.status_ok => error.login.failed
1714947444139	Sync.SyncAuthManager	ERROR	Failed to fetch a token for authentication: TokenServerClientNetworkError({"error":{}})(resource://services-common/tokenserverclient.sys.mjs:28:36) JS Stack trace: TokenServerClientNetworkError@tokenserverclient.sys.mjs:51:16
_tokenServerExchangeRequest@tokenserverclient.sys.mjs:235:13
1714947444142	Services.Common.RESTRequest	DEBUG	GET https://profile.accounts.firefox.com/v1/profile failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1714947444143	FirefoxAccounts	ERROR	Background refresh of profile failed: {"name":"FxAccountsProfileClientError","code":null,"errno":998,"error":"NETWORK_ERROR","message":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.sys.mjs :: onStopRequest :: line 478\"  data: no]"}
1714947444143	FirefoxAccounts	ERROR	Background refresh of profile failed: {"name":"FxAccountsProfileClientError","code":null,"errno":998,"error":"NETWORK_ERROR","message":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.sys.mjs :: onStopRequest :: line 478\"  data: no]"}
1714947444160	Sync.Resource	WARN	GET request to https://sync-1-us-west1-g.sync.services.mozilla.com/1.5/204028242/info/collections failed: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1714947444160	Sync.Status	DEBUG	Status.sync: success.sync => error.login.reason.network
1714947444160	Sync.Status	DEBUG	Status.service: error.login.failed => error.sync.failed
1714947444160	Sync.ErrorHandler	ERROR	Sync encountered an error: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1714947444160	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms. (why=reschedule)
1714947444161	Sync.Telemetry	INFO	Early submission of sync telemetry due to changed IDs/NodeType
1714947444165	Sync.Service	DEBUG	Exception calling anonymous function: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
